### DISCUSSION TITLE
Move @ryanmoran to an Emeritus Position in the Tooling subteam

### DISCUSSION THREAD

**Discussion by sophiewigmore** [paketo-buildpacks/feedback#239] [Tooling Team]
I would like to propose we move @ryanmoran into an Emeritus Maintainer position for the Tooling subteam. He is moving away from the project after a number of years building up the technology and community. Thank you for everything you've done for the project, Ryan!

@paketo-buildpacks/tooling-maintainers please vote as to whether we should officially make this change
cc @paketo-buildpacks/steering-committee

**Comment by ForestEckhardt** [comment]
@ryanmoran has been removed from the Tooling Maintainers team.

Please open a PR on the [TEAMS.md](https://github.com/paketo-buildpacks/community/blob/main/TEAMS.md) to remove @ryanmoran  to the list of maintainers.
